class.jetpack.php

```diff
@@ -537,7 +537,7 @@ private function __construct() {
 		 * Prepare Gutenberg Editor functionality
 		 */
 		require_once JETPACK__PLUGIN_DIR . 'class.jetpack-gutenberg.php';
-		add_action( 'enqueue_block_assets', array( 'Jetpack_Gutenberg', 'enqueue_block_assets' ) );
+		add_action( 'init', array( 'Jetpack_Gutenberg', 'load_blocks' ), 1000 );
```

Comment:
Why the big priority here?

Reply:
The reason for it is to make sure that modules are loaded before the blocks are registered. 
the default value should be fine since modules are loaded before init anyways.